packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -715,6 +712,19 @@ class RenderEditable extends RenderBox {
     markNeedsSemanticsUpdate();
   }
 
+  /// {@template flutter.widgets.editable.selectionEnabled}
+  /// Decides if selection is enabled based on [enableInteractiveSelection] and
+  /// [obscureText] to make the common use case of password fields simple while
+  /// still allowing selection of obscured text.
+  ///
+  /// Just setting [obscureText] to true will create a typical Material password
```

Comment:
Don't refer to Material here, RenderEditable and EditableText are used elsewhere.  The simplest way to define this method's value might be to do the case analysis:

- By default [enableInteractiveSelection] is null, obscureText is false, and this method returns true.
- If [enableInteractiveSelection] is null, obscureText is true, then this method returns false. This is the common case for password fields.
- If  [enableInteractiveSelection] is non-null then its value is returned.  An app might set it to true to enable interactive selection for a password field, or to false to unconditionally disable interactive selection.